sample_consensus/include/pcl/sample_consensus/impl/sac_model_cylinder.hpp

```diff
@@ -321,7 +320,6 @@ pcl::SampleConsensusModelCylinder<PointT, PointNT>::projectPoints (
   Eigen::Vector4f line_dir (model_coefficients[3], model_coefficients[4], model_coefficients[5], 0.0f);
   float ptdotdir = line_pt.dot (line_dir);
   float dirdotdir = 1.0f / line_dir.dot (line_dir);
```

Comment:
```suggestion
  float dirdotdir = 1.0f / line_dir.dot (line_dir);

```
Undo the line removed here